app/controllers/api/v1/podcasts_controller.rb

```diff
@@ -15,7 +16,12 @@ def create
     end
 
     def show
-        render json: Podcast.find(params[:id])
+        podcast = Podcast.find(params[:id])
+        user = current_user
+        response_body = []
+        response_body.push(user)
+        response_body.push(podcast)
+        render json: response_body
     end
```

Comment:
wooohhh lets slowdown here. Ideally we dont create a response body that is just a nameless array. If you wanted to get this working, it would be better to create a hash, had a user and podcast key, add the values in for those respectively, and then return the response. 

_However...._ all of this work is just putting off the inevitable: adding a Podcast serializer. In your podcast serializer, you can scope in the current user so that `render json: Podcast.find(params[:id])` would work once again. In fact, this file will be ready to go once your show method only contains `render json: Podcast.find(params[:id])` once again (after adding a serializer. 

Get to it ;)